src/styles/introduction.module.css

```diff
@@ -0,0 +1,17 @@
+@import url('https://fonts.googleapis.com/css?family=Lilita+One|Quicksand:300&display=swap');
+
+ .container {
+    position: relative;
+ }
+
+
+.font {
+    font-family: 'Quicksand', sans-serif;
+    position: absolute;
+    top: 45%;
```

Comment:
To center an element, there is probably a better way.
As yours is not flexible enough for changes like text, font-weight or font-size for example.